api/novelist.py

```diff
@@ -39,17 +39,28 @@ class NoveListAPI(object):
     # from the database.
     QUERY_ENDPOINT = "http://novselect.ebscohost.com/Data/ContentByQuery?\
             ISBN=%(ISBN)s&ClientIdentifier=%(ClientIdentifier)s&version=%(version)s"
-    MAX_REPRESENTATION_AGE = 6*30*24*60*60      # six months
+    MAX_REPRESENTATION_AGE = 14*24*60*60      # two weeks
```

Comment:
I brought this down a lot because we're now using it for recommendations, instead of less volatile background data like equivalent ISBNs.